### DISCUSSION TITLE
Why GitHub Stars List couldn't no longer be created?

### DISCUSSION THREAD

**Discussion by ToNyRANDRIAMANANTSOA** [community/community#42823] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I used to use GitHub Stars List to categorize my favorite projects. It was a very helpful feature that allows me to easily find the right project later in my hundreds or thousands of starred repos.
Unfortunately, I could not recently create a new list because the button is grayed out. Does it mean that it has a limited number of items or that the feature is gone?

Screenshot: 
![image](https://user-images.githubusercontent.com/50623932/209916284-85345f1c-871c-45ae-85d8-2a7afec6aabe.png)

Thank you.

**Comment by radar-blip** [comment, marked as answer]
I think it is limited (i've seen people mention it before) but I'm not sure what the limit is.
You could try deleting a few unimportant lists and re-creating others.

According to these discussions #8633, the limit of lists was once 16, then they doubled it to 32.

**Reply by ToNyRANDRIAMANANTSOA** [reply]
Alright, thank you for the information, hope they will make it unlimited as 32 might even not be enough as projects are growing.

**Reply by radar-blip** [reply]
Right, lots of people agree 32 isn't enough either.

**Reply by ramazansancar** [reply]
I agree. I am in the same situation.

**Reply by Phuchong0259** [reply]
ผมอ่านไม่ออก🥹

**Comment by fglueck** [comment]
I have 17 entries in my list and got the error message that I can only 32. 
But 32-17 = 15.
Where are the missing 15 ?
So I think there is a bug.

**Reply by Saud11ems** [reply]
@Bbmccmvcx

**Comment by Minou87** [comment]
```

```

**Comment by Adawntoremember** [comment]
I second that hahah

**Comment by Phuchong0259** [comment]
เน็ตผมยัง9บาทต่อวันตังยังไม่มีเติมเลยครับงานก็ไม่ได้ทำ

ในวันที่ จ. 12 ก.พ. 2024 10:47 น. dandan3100 ***@***.***>
เขียนว่า:

**Comment by Saud11ems** [comment]
@ToNyRANDRIAMANANTSOA

**Comment by Phuchong0259** [comment]
It works! ผมอยากถอนเงินสมแล้วถอนได้ไหม

ในวันที่ พฤ. 15 ก.พ. 2024 18:01 น. Saud11ems ***@***.***>
เขียนว่า:
